docker-mixin/config.libsonnet

```diff
@@ -0,0 +1,8 @@
+{
+  _config+:: {
+    dashboardTags: ['docker-integration'],
+    dashboardPeriod: 'now-1h',
+    dashboardTimezone: 'default',
+    dashboardRefresh: '1m',
+  },
```

Comment:
Just as a heads-up, these things are all part of the "prepare dashboards" functionality that is executed against all mixins when they become grafana cloud integrations.

It's fine to have these templatized for non grafana cloud users, tho I would probably choose a tag other than `docker-integration`. Perhaps just `docker`?

Reply:
Thanks! Changed to "docker"